bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/selector/ScheduleEditor.scala

```diff
@@ -245,8 +252,8 @@ class ScheduleEditor(ctx: QvContext, scheduleCache: ScheduleCache) extends Dialo
       start <- cd.startDate
       end <- cd.endDate
     } yield {
-      val s = start + TimeUtils.hours(14)
-      val e = end + TimeUtils.hours(14) + TimeUtils.days(2)
+      val s = start + 14.hours.toMillis
```

Comment:
What's the 14 hours? Does it mean 2PM as the time our day shifts?
Could we use a time based type rather than millis?

Reply:
I think the 14 hours refers to the start of day hour as per later on.
You're right that it should be changed to use the constant.
There are so many repeated constants.

Reply:
Seems like this will be wrong twice a year at the DST switchover because the observing day changes at 2pm *local* time … you can't express it in epoch time. If this is the case maybe open another issue, it seems minor.